add_book.php

```diff
@@ -0,0 +1,34 @@
+<?php
+include 'database.php';
+
+if ($_SERVER['REQUEST_METHOD'] == 'POST') {
```

Comment:
Observations: SQL Injection vulnerability due to directly embedding user input into SQL queries.
Recommendations: Use prepared statements to safely handle user input and prevent SQL injection.

```php
$stmt = $connection->prepare("INSERT INTO books (title, author, year) VALUES (?, ?, ?)");
$stmt->bind_param('ssi', $title, $author, $year);
$stmt->execute();
$stmt->close();
```